src/main/kotlin/app/revanced/patches/youtube/shorts/overlaybuttons/ShortsOverlayButtonsPatch.kt

```diff
@@ -0,0 +1,86 @@
+package app.revanced.patches.youtube.shorts.overlaybuttons
+
+import app.revanced.patcher.data.ResourceContext
+import app.revanced.patcher.patch.PatchException
+import app.revanced.patcher.patch.options.PatchOption.PatchExtensions.stringPatchOption
+import app.revanced.patches.youtube.utils.compatibility.Constants.COMPATIBLE_PACKAGE
+import app.revanced.patches.youtube.utils.settings.SettingsPatch
+import app.revanced.util.ResourceGroup
+import app.revanced.util.copyResources
+import app.revanced.util.patch.BaseResourcePatch
+
+@Suppress("unused")
+object ShortsOverlayButtonsPatch : BaseResourcePatch(
+    name = "Shorts overlay buttons",
```

Comment:
The `Overlay buttons` patch adds new player overlay buttons not provided by YouTube - such as Copy video URL and Whitelist.

The patch name `Shorts overlay buttons` can be confusing as it seems to be adding new Shorts overlay buttons that are not provided by YouTube.

The patch name should be changed to something like `Shorts action buttons theme`

Reply:
What about `Custom Shorts action buttons`?

Reply:
> What about `Custom Shorts action buttons`?

This one is good and consistent with similar patches. 

I named it like that because `Overlay buttons` also included Fullscreen button which already was there, but I'm fine with anything.